package.json

```diff
@@ -19,7 +19,8 @@
   },
   "license": "MIT",
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=10.0.0",
```

Comment:
@poveden This change has been driven by the `node-jose` upgrade (cf. https://github.com/cisco/node-jose/blob/master/CHANGELOG.md#200-2020-09-20)